File: src/Paiza/D100.kt
package Paiza

fun main() {
    val words: String = readLine()!!
    val ch1: Char = '-'
    val ch2: Char = '_'

    var i = 0
    var count1 = 0 //카운트
    var count2 = 0 //카운트

    while (i < words.length) {
        if (ch1 == words[i]) {
            count1 += count1 + 1
        }else if (ch2 == words[i]){
            count2 += count2 + 1
        }
        i++

    }

    if (count1 > count2){
        println(words.replace("_","-"))
    }else{
        println(words.replace("-","_"))
    }
}

File: src/Paiza/python/D100.py
# 100점
sc = input()

ch1 = '_'
ch2 = '-'

count1 = 0
count2 = 0

for i in sc:
    if i == ch1:
        count1 += 1
    elif i == ch2:
        count2 += 1

if count1 > count2:
    print(sc.replace('-','_'))
elif count2 > count1:
    print(sc.replace('_','-'))
else:
    print(sc.replace('-','_'))
